src/coreclr/src/librariesshims/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+set(EXCLUDE_VERSION_HEADER_FILE ON)
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+if(CLR_CMAKE_HOST_UNIX AND NOT DEFINED CLR_CROSS_COMPONENTS_BUILD)
+    set(BUILD_GLOBALIZATION_SHARED_LIBRARY OFF)
+    include_directories("../../../libraries/Native/Unix/Common")
```

Comment:
Why do you need this include here? Would it be better for `System.Globalization.Native` to have it inside?

Reply:
Because `src/libraries/Native/Unix` adds that include and we don't include that file on this path. However, I could set a variable and then condition the include in `System.Globalization.Native` when that variable is ON